targets/ipq807x-generic

```diff
@@ -0,0 +1,11 @@
+-- Xiaomi
+
+device('xiaomi_ax3600', 'xiaomi-ax3600', {
```

Comment:
```suggestion
device('xiaomi-ax3600', 'xiaomi_ax3600', {
```
Mistake on my end. As @maurerle explained this should be the other way arround.

Reply:
<img width="499" alt="image" src="https://github.com/user-attachments/assets/f5fae2d2-6d90-4a4c-a696-849eeff1c701">

Der Model String im Device Tree: xiaomi-ax3600. Screenshot zur Dokumentation.